### PR TITLE
[serving] Improve logging for exec shell command

### PR DESCRIPTION
**Pull request by xyang16**
## Description ##

Brief description of what this PR is about

Improve logging for exec shell command

### CODE REVIEW

**Comment by xyang16** [comment]
Raising PR on behalf of Frank.

**Review by frankfliu — APPROVED** [review]

File: wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java [hunk 1]
```diff
@@ -54,6 +52,21 @@ public final class LmiUtils {
 
     private LmiUtils() {}
 
+    static void exec(List<String> cmd) throws IOException, InterruptedException {
```
Comment (siddvenk) [inline wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java:55]: We can do this as a follow up, but might be nice to put this into a generic ProcessUtils class or something similar. We have a few more places where we do this (like cluster request handler)

File: wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java [hunk 2]
```diff
@@ -54,6 +52,21 @@ public final class LmiUtils {
...
+        Process exec = new ProcessBuilder(cmd).redirectErrorStream(true).start();
+        String logOutput;
+        try (InputStream is = exec.getInputStream()) {
+            logOutput = Utils.toString(is);
+        }
+        int exitCode = exec.waitFor();
+        if (0 != exitCode || logOutput.startsWith("ERROR ")) {
+            logger.info("{}", logOutput);
```
Comment (siddvenk) [inline wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java:63]: should this be logged at error level?

File: wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java (hunk 1 above)
Comment (frankfliu) [reply wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java:55]: We can make this in `ProcessUtils` class, but need to make error checking more generic, this class currently has logic specific for s3cmd:

```
logOutput.startsWith("ERROR ")
```

We can pass a regex if caller need extra error checking:
```
(regex != null && logOutput.matches(regex))
```

**Review by siddvenk — APPROVED** [review]

**Pull request merged**